examples/access_generator/main.py

```diff
@@ -0,0 +1,115 @@
+from typing import Iterable, Tuple, Optional
+
+from devana.code_generation.printers.default.defaultprinter import create_default_printer, CodePrinter
+from devana.syntax_abstraction.attribute import Attribute, AttributeDeclaration
+from devana.syntax_abstraction.classinfo import MethodInfo, Comment, FunctionInfo, AccessSpecifier, ClassInfo, FieldInfo
+from devana.syntax_abstraction.functioninfo import BasicType, FunctionModification
+from devana.syntax_abstraction.typeexpression import TypeModification
+from devana.syntax_abstraction.organizers import *
+
+
+def get_devana_comments(comment: Comment) -> Iterable[str]:
+    # Function to get all devana comments properly formatted.
+    for comment_source in comment.text:
+        if (formatted_comment := comment_source.lstrip()).startswith("devana:"):
+            yield formatted_comment
+
+
+def get_field_name(field: FieldInfo) -> str:
+    # Function to retrieve the field name.
+    name: str = field.name.capitalize()
+
+    if field.associated_comment:
+        for text in get_devana_comments(field.associated_comment):
+            if text.startswith("devana: custom-name="):
+                name = text.split("=")[1]
+                break
+
+    return name
+
+
+def create_field_methods(field: FieldInfo) -> Tuple[MethodInfo, MethodInfo]:
+    # Function to generate setter and getter for field.
+    def should_ignore_attributes(comment: Optional[Comment]) -> bool:
+        return "devana: ignore-attributes" in get_devana_comments(comment) if comment else False
+
+    field_name: str = get_field_name(field)
+    maybe_unused_attribute: Attribute = Attribute("maybe_unused")
+    nodiscard_attribute: Attribute = Attribute("nodiscard")
+
+    field_setter: MethodInfo = MethodInfo()
+    field_setter.body = f"{field.name} = new{field.name.capitalize()};"
+    field_setter.access_specifier = AccessSpecifier.PUBLIC
+    field_setter.name = f"set{field_name}"
+    field_setter.return_type = BasicType.VOID
+
+    setter_arg: FunctionInfo.Argument = FunctionInfo.Argument()
+    setter_arg.name = f"new{field.name.capitalize()}"
+    setter_arg.type.details = field.type
+    if field.type.name == "string":
+        setter_arg.type.modification |= TypeModification.CONST | TypeModification.REFERENCE
+
+    field_setter.arguments.append(setter_arg)
+
+    field_getter: MethodInfo = MethodInfo()
+    field_getter.body = f"return {field.name};"
+    field_getter.access_specifier = AccessSpecifier.PUBLIC
+    field_getter.name = f"get{field_name}"
+    field_getter.return_type = field.type
+    field_getter.modification |= FunctionModification.CONST
+
+    if not should_ignore_attributes(field.associated_comment):
+        field_getter.attributes.append(AttributeDeclaration([nodiscard_attribute, maybe_unused_attribute]))
+        field_setter.attributes.append(AttributeDeclaration([maybe_unused_attribute]))
+
+    return field_setter, field_getter
+
+
+def create_class_methods(class_info: ClassInfo) -> None:
```

Comment:
When a function returns nothing, it is customary to not use None as a hint about the return type.  Please, remove the annotations.
For the future, None is used for something else, such as a union or an optional. To describe functions that return nothing uses:
> from typing import NoReturn
> def foop() -> NoReturn:

Reply:
Z tego co wiem, to `NoReturn` jest stosowane tylko w sytuacjach, gdy funkcja faktycznie nic nie zwraca (np. Wywala wyjątek). Takie przypadki są rzadkie, a funkcje domyślnie zwracają `NoneType`.

Reply:
As far as I know, `NoReturn` is only used in situations where the function actually returns nothing (e.g., it throws an exception). Such cases are rare, and functions return `NoneType` by default, in this case also.

Reply:
Yes, you're right - it was a side note before such hints.
We don't want None anyway: https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/useless-return.html

Reply:
Done in https://github.com/JhnW/devana/pull/16/commits/6cb2adf143e551c5d92e1a76e853e6d2389be2c3